lib/cli.js

```diff
@@ -396,14 +396,16 @@ exports.run = async () => { // eslint-disable-line complexity
 			projectDir,
 			reportStream: process.stdout,
 			stdStream: process.stderr,
-			watching: combined.watch
+			watching: combined.watch,
+			mode: 'verbose'
 		});
 	} else {
 		reporter = new MiniReporter({
 			projectDir,
 			reportStream: process.stdout,
 			stdStream: process.stderr,
-			watching: combined.watch
+			watching: combined.watch,
+			mode: 'verbose'
```

Comment:
This shouldn't be verbose.

Let's see if we can't merge the `if` / `else` into a single instantiation.